src/benchmarklib/abstract_benchmark_item_runner.hpp

```diff
@@ -35,6 +35,9 @@ class AbstractBenchmarkItemRunner {
   // Returns the BenchmarkItemIDs of all selected items
   virtual const std::vector<BenchmarkItemID>& items() const = 0;
 
+  // Loads the dedicated expected restuls as tables into _dedicated_expected_results
+  virtual void load_dedicated_expected_results();
```

Comment:
How does this differ between different implementation of the AbstractBenchmarkItemRunner, i.e., why is it virtual?